multinode/bandwidth/service.go

```diff
@@ -105,12 +105,15 @@ func (service *Service) MonthlySatellite(ctx context.Context, satelliteID storj.
 	cache := make(UsageRollupDailyCache)
 
 	for _, node := range listNodes {
+		nodeinfo := service.nodes.FetchNodeInfo(ctx, node)
```

Comment:
It's a bit overkill to fetch all the node info only to check if it's online.
We could track the check from `FetchNodeInfo` to a separate function and call it from there and here.